python_modules/dagster/dagster_tests/execution_tests/versioning_tests/test_data_versions.py

```diff
@@ -1051,5 +1051,5 @@ def downstream_asset(**kwargs):
     materialize_assets(all_assets, instance)[downstream_asset.key]
     assert traced_counter.get().counts() == {
         "DagsterInstance.get_asset_records": 1,
-        "DagsterInstance.get_run_record_by_id": 1,
+        "DagsterInstance.get_run_record_by_id": 3,
```

Comment:
i think this is bumped up because we call `get_run_by_id` in `handle_new_event` now

Reply:
might be worth adding a comment